File: templates/Uwp/Features/DeepLinking.MVVMLight/ViewModels/ViewModelLocator_postaction.cs
namespace Param_RootNamespace.ViewModels
{
    public class ViewModelLocator
    {
        private ViewModelLocator()
        {
            //^^
            //{[{
            Register<SchemeActivationSampleViewModel, SchemeActivationSamplePage>();
            //}]}
        }

        //{[{
        public SchemeActivationSampleViewModel SchemeActivationSampleViewModel => SimpleIoc.Default.GetInstance<SchemeActivationSampleViewModel>();
        //}]}
    }
}

File: templates/Uwp/Features/DeepLinking.MVVMLightVB/ViewModels/ViewModelLocator_postaction.vb
Namespace ViewModels
    Public Class ViewModelLocator
        Private Sub New()
            '^^
            '{[{
            Register(Of SchemeActivationSampleViewModel, SchemeActivationSamplePage)()
            '}]}
        End Sub

'{[{

        Public ReadOnly Property SchemeActivationSampleViewModel As SchemeActivationSampleViewModel
            Get
                Return SimpleIoc.[Default].GetInstance(Of SchemeActivationSampleViewModel)()
            End Get
        End Property
'}]}
    End Class
End Namespace
